packages/diagrams/backend/sirius-components-collaborative-diagrams/src/test/java/org/eclipse/sirius/components/collaborative/diagrams/handlers/GetConnectorToolsEventHandlerTests.java

```diff
@@ -171,6 +172,7 @@ public void testGetConnectorTools() {
                 .edgeDescriptions(new ArrayList<>())
                 .palettes(List.of(palette))
                 .dropHandler(variableManager -> new Failure(""))
+                .styleProvider(variableManager -> DiagramStyle.newDiagramStyle().build())
```

Comment:
Any reason to have some of the `DiagramStyle` in the tests with the default (transparent) background and others with `.bkacground("#FAFAFA")` like the one below in the same file?

Reply:
No, in fact, I started by adding a value at instantiation and then adding a default value.
I changed to use the default value in the tests